openedx/openedx-platform
# fix: Refactor xss-lint exceptions that break the certificate share-in-facebook link

## Description

The certificate banner template has "## xss-lint" statements that break the share in facebook link

## Testing instructions

- [Enable social sharing for certificates](https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/open-release-maple.master/configuration/enable_socialsharing_icons.html#enabling-social-sharing-of-courses-and-certificates)
- Preview a certificate
- Click on the facebook link
- A pop up window should show up to share in facebook

## Other information

Please check if this impacts any lint check.
